pkg/qdr/qdr.go

```diff
@@ -501,13 +505,14 @@ type Address struct {
 
 type TcpEndpoint struct {
 	ExtraFieldsImpl
-	Name           string `json:"name,omitempty"`
-	Host           string `json:"host,omitempty"`
-	Port           string `json:"port,omitempty"`
-	Address        string `json:"address,omitempty"`
-	SiteId         string `json:"siteId,omitempty"`
-	SslProfile     string `json:"sslProfile,omitempty"`
-	VerifyHostname *bool  `json:"verifyHostname,omitempty"`
+	Name                     string `json:"name,omitempty"`
+	Host                     string `json:"host,omitempty"`
+	Port                     string `json:"port,omitempty"`
+	Address                  string `json:"address,omitempty"`
+	SiteId                   string `json:"siteId,omitempty"`
+	SslProfile               string `json:"sslProfile,omitempty"`
+	VerifyHostname           *bool  `json:"verifyHostname,omitempty"`
+	CloseConnectionsOnDelete *bool  `json:"closeConnectionsOnDelete,omitempty"`
```

Comment:
Could the type be `bool` instead of `*bool` ?  That would simplify the rest of the code.

Reply:
Ah, I see that I misread the corresponding skupper-router PR.  

When I saw this line in the skupper-router PR, I (incorrectly) assumed that the new optional flag was a bool pointer.
My mistake.
` config->terminate_conns   = qd_entity_opt_bool(entity, "closeConnectionsOnDelete", false);`
 
I will update to change CloseConnectionsOnDelete to be a bool.